trade-finance-manager-api/src/v1/helpers/amendment.helpers.api-test.js

```diff
@@ -0,0 +1,291 @@
+        conditions: amendmentVariables.approvedWithWithoutConditionsBothAmendments.amendment.ukefDecision.conditions,
+      },
+    );
+
+    expect(updateFacilityAmendmentSpy).toHaveBeenCalledWith(
+      amendmentVariables.approvedWithWithoutConditionsBothAmendments.facilityId,
+      amendmentVariables.approvedWithWithoutConditionsBothAmendments.amendmentId,
+      { ukefDecision: { managersDecisionEmailSent: true } },
+    );
+  });
+
+  it('should send approved with conditions declined email with correct details for both amendments being approved w/ conditions and declined', async () => {
+    await sendManualDecisionAmendmentEmail(amendmentVariables.approvedWithConditionsDeclined);
+
+    expect(sendEmailApiSpy).toHaveBeenCalledWith(
+      CONSTANTS.EMAIL_TEMPLATE_IDS.MANUAL_AMENDMENT_DECISION_APPROVED_W_CONDITIONS_DECLINED,
+      amendmentVariables.approvedWithConditionsDeclined.user.email,
+      {
+        bankReferenceNumber: amendmentVariables.approvedWithConditionsDeclined.dealSnapshot.bankInternalRefName,
+        exporterName: amendmentVariables.approvedWithConditionsDeclined.dealSnapshot.exporter.companyName,
+        recipientName: 'Bob Smith',
+        ukefDealId: amendmentVariables.approvedWithConditionsDeclined.dealSnapshot.ukefDealId,
+        ukefFacilityId: amendmentVariables.approvedWithConditionsDeclined.amendment.ukefFacilityId,
+        conditions: amendmentVariables.approvedWithConditionsDeclined.amendment.ukefDecision.conditions,
+        declined: amendmentVariables.approvedWithConditionsDeclined.amendment.ukefDecision.declined,
+        amendmentTypeApproved: CONSTANTS.DEALS.AMENDMENT_TYPE.VALUE,
+        amendmentTypeDeclined: CONSTANTS.DEALS.AMENDMENT_TYPE.COVER_END_DATE,
+      },
+    );
+
+    expect(updateFacilityAmendmentSpy).toHaveBeenCalledWith(
+      amendmentVariables.approvedWithConditionsDeclined.facilityId,
+      amendmentVariables.approvedWithConditionsDeclined.amendmentId,
+      { ukefDecision: { managersDecisionEmailSent: true } },
+    );
+  });
+
+  it('should send approved with conditions declined email with correct details for both amendments being approved w/ conditions and declined', async () => {
+    await sendManualDecisionAmendmentEmail(amendmentVariables.approvedWithConditionsDeclinedSwapped);
+
+    expect(sendEmailApiSpy).toHaveBeenCalledWith(
+      CONSTANTS.EMAIL_TEMPLATE_IDS.MANUAL_AMENDMENT_DECISION_APPROVED_W_CONDITIONS_DECLINED,
+      amendmentVariables.approvedWithConditionsDeclinedSwapped.user.email,
+      {
+        bankReferenceNumber: amendmentVariables.approvedWithConditionsDeclinedSwapped.dealSnapshot.bankInternalRefName,
+        exporterName: amendmentVariables.approvedWithConditionsDeclinedSwapped.dealSnapshot.exporter.companyName,
+        recipientName: 'Bob Smith',
+        ukefDealId: amendmentVariables.approvedWithConditionsDeclinedSwapped.dealSnapshot.ukefDealId,
```

Comment:
for these Ids, you can use something like:
`ukefDealId: expect.any(String)`
this because we know that it has to have a value, and that value needs to be a String

Reply:
same thing applies to the other blocks